python/distant_vfx/jobs/chunk_to_new_packages.py

```diff
@@ -26,11 +26,12 @@ def main(dest_dir, paths, should_move=False):
                     cmd = ['cp', '-r', item, new_package_path]
                 else:
                     cmd = ['mv', item, new_package_path]
-                result = subprocess.run(cmd,
+                process = subprocess.Popen(cmd,
```

Comment:
There is no real point to use `subprocess.run` ever, and it doesn't exist in python2.

Use `subprocess.Popen` and then call `process.communicate` to get the stdout and stderr.

Reply:
The `communicate` call waits for the process to finish and the `run` call hides that from you

Reply:
Gotcha, good tip, thanks!